src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
```

Comment:
alert --> alerts

Reply:
I feel like you threw the baby out with the bath water here :)

I'd love to see a combo of what was here before and the new thing you've got. (Hint: I like the first part of your sentence and the second part of the old.)